pittAPI.py

```diff
@@ -495,7 +495,10 @@ def get_dining_locations_by_status(self, status=None):
         return dining_locations
 
     def get_dining_location_by_name(self, location):
-        return self.get_dining_locations()[self._encode_dining_location(location)]
+        try:
+            return self.get_dining_locations()[self._encode_dining_location(location)]
+        except:
+            return {}
```

Comment:
Should I throw a custom exception? -- oh wait we have the 'InvalidParameterException'